Fix 'quivalent' typo in pharo --help
--- a/platforms/unix/vm/sqUnixMain.c
+++ b/platforms/unix/vm/sqUnixMain.c
@@ -1477,12 +1477,12 @@
   printf("  --jit                  enable the dynamic compiler (if available)\n");
 # endif
   printf("  --notimer              disable interval timer for low-res clock\n");
-  printf("  --display <dpy>        quivalent to '-vm-display-X11 --display <dpy>'\n");
-  printf("  --headless             quivalent to '-vm-display-X11 --headless'\n");
-  printf("  --nodisplay            quivalent to '-vm-display-null'\n");
+  printf("  --display <dpy>        equivalent to '-vm-display-X11 --display <dpy>'\n");
+  printf("  --headless             equivalent to '-vm-display-X11 --headless'\n");
+  printf("  --nodisplay            equivalent to '-vm-display-null'\n");
   printf("  --nomixer              disable modification of mixer settings\n");
-  printf("  --nosound              quivalent to '-vm-sound-null'\n");
-  printf("  --quartz               quivalent to '-vm-display-Quartz'\n");
+  printf("  --nosound              equivalent to '-vm-sound-null'\n");
+  printf("  --quartz               equivalent to '-vm-display-Quartz'\n");
 #endif
 }
 
